Magicnumbers/sandbox-mode.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+#include <ctime>
+#include <Windows.h>
+#include <conio.h>
+using namespace std;
+
+void sandboxMode() {
+
+	srand(static_cast<unsigned int>(time(nullptr)));
```

Comment:
i like the use of srand, it prevents the program from making you guess the same number every time